### ISSUE TITLE
Icon files generated, but do not appear in manifest or favicon

### ISSUE THREAD

**Issue by tmorehouse** [nuxt-community/pwa-module#231]
### Version

[v3.0.0-beta.17](https://github.com/pwa-module/releases/tag/v3.0.0-beta.17)

### Reproduction link

[https://deploy-preview-4028--bootstrap-vue.netlify.com/](https://deploy-preview-4028--bootstrap-vue.netlify.com/)

### Steps to reproduce

Inspect `<head>` element and see that there is no `favicon`.  Inpsect manifest to see that icons is an empty array.

### What is expected ?

Faviocon is generated, manifest populated with icons array

### What is actually happening?

no icons (although Icon files are generated in the /icons directory)

/cc @pi0 

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/pwa-module/issues/c167">#c167</a>)</em></sub></div>

**Comment by tmorehouse** [comment]
Were were using the defaults (default icon file and location, etc), and see the icon files being generated.

Added in new `pwa: { icons: { ...} }` icon entry into `nuxt.config.js`, but see the same results.

**Comment by preventdefault** [comment]
Hi,
after digging into the source code if found a little workaround for that:
```
manifest: {
    icons: [],
  },
  pwa: {
    icon: {
      iconFileName: 'android-chrome-512x512.png',
      sizes: [36, 48, 72, 96, 144, 192, 256, 384, 512],
    },
    manifest: {
      name: 'Foo',
      short_name: 'Foo',
    },
  },
```

**Comment by tmorehouse** [comment]
I've verified that @preventdefault's workaround works

**Comment by pi0** [comment]
Fixed by [v3.0.0-beta.18](https://github.com/nuxt-community/pwa-module/releases/tag/v3.0.0-beta.18)

**Comment by moneymc** [comment]
It doesn't work on v3.0.0-beta.19
